### PR TITLE
Updated readme with more details and instructions.

### PR DESCRIPTION
**Pull request by mohlsen**
Finishes #1 and should be sufficient for #27.  Should also be enough to do #29

### CODE REVIEW

**Comment by mohlsen** [comment]
@YashdalfTheGray @kwpeters can one of you review these readme changes?  Added some more examples and a bit of documentation.  Wanted this before publishing on npm.

File: README.md [hunk 1]
```diff
@@ -23,28 +23,73 @@ Currently Supporting:
...
 
-### Usage
+## Usage
+
+### CLI
+
+#### Install
+Can be installed globally or in a local directory.
+
+- **Globally**: `npm install -g https://github.com/mohlsen/check-engine.js.git`
```
Comment (YashdalfTheGray) [inline README.md]: Should the `.js` still be there close to the end of the git install path here and on the next line?

File: README.md [hunk 2]
```diff
@@ -23,28 +23,73 @@ Currently Supporting:
...
+- **Local**: `npm install https://github.com/mohlsen/check-engine.js.git`
+
+#### Running:
+1. Run `check-engine [path_to_package.json]`. `check-engine` located in `/bin/check-engine` if using a local install.
```
Comment (YashdalfTheGray) [inline README.md]: I think "...`check-engine` is located in `/bin/check-engine`..." Sounds a little better.

File: README.md [hunk 3]
```diff
@@ -23,28 +23,73 @@ Currently Supporting:
...
+ - if specified, [path_to_package.json], is the location of a `package.json` file.
```
Comment (YashdalfTheGray) [inline README.md]: What do you think about "if specified, [path_to_package.json] is the location of a package.json file for check-engine to validate against."?

**Review by YashdalfTheGray — DISMISSED** [review]
Pretty much all looks good except for a couple of wording changes.

File: README.md (hunk 3 above)
Comment (YashdalfTheGray) [inline README.md]: I guess `check-engine` is validating against that file so I think this sounds better, "if specified, [path_to_package.json] is the location of a package.json file for check-engine to validate against."

**Review by mohlsen — COMMENTED** [review]
all updated

**Pull request merged**